test/explicit-resource-management/main.test.ts

```diff
@@ -0,0 +1,120 @@
+
+import { expect } from 'chai';
+import { describe, it } from 'mocha';
+import { AbstractCursor, ChangeStream, ClientSession, GridFSBucket, MongoClient } from 'mongodb/lib/beta';
+import * as sinon from 'sinon';
+import { Readable } from 'stream';
+import { pipeline } from 'stream/promises';
+import { setTimeout } from 'timers/promises';
+
+async function setUpCollection(client: MongoClient) {
+	const collection = client.db('foo').collection<{ name: string }>('bar');
```

Comment:
it looks like this file is using tab-formatting which makes it look different than the other test file in this PR (and in the code base) - can we update it to be consistent with the other files?